db/schema.rb

```diff
@@ -53,6 +53,7 @@
     t.datetime "wentLiveAt", precision: nil
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
+    t.datetime "closedOnDate", null: false
```

Comment:
You can't make changes directly to the schema file, you need to create a migration in order to do this. https://edgeguides.rubyonrails.org/active_record_migrations.html
There is a file in rails called under `db/migrate` where it will show you all the migrations we have created to make database changes. 
This one in particular shows how to add columns to a table `db\migrate\20220317174955_add_order_to_tables.rb`
Once you have made the migration you then have to run the `rake db:migrate` command for this change here to take place.

Reply:
one sec lemme find another pr as an example

Reply:
https://github.com/AashnaNarang/SYSC4806/pull/39

In this pr I added the order columns to some tables

To create the migration there is a command you can run

it will be something like

`rails g migration add_insert_column_name_here_to_table_name`

This will make the migration file

You can then add in the changes to the schema you want to do in there